extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -170,6 +176,8 @@ class AmpAccordion extends AMP.BaseElement {
         header.setAttribute('tabindex', 0);
       }
       this.headers_.push(header);
+      content.setAttribute('aria-labelledby', headerId);
```

Comment:
Do these values need to be set after the `push` above or should they be set beforehand?

Reply:
They can be set before or after the `push`.  I just figured I'd keep all the `header` related lines together than do the `content` after.